tests/test_models_bert.py

```diff
@@ -12,87 +12,88 @@ def test_list_pretrained_bert():
 
 
 @pytest.mark.parametrize('compute_layout', ['auto', 'NT', 'TN'])
-def test_bert_small_cfg(compute_layout):
-    cfg = BertModel.get_cfg()
-    cfg.defrost()
-    cfg.MODEL.vocab_size = 100
-    cfg.MODEL.units = 12 * 4
-    cfg.MODEL.hidden_size = 64
-    cfg.MODEL.num_layers = 2
-    cfg.MODEL.num_heads = 2
-    cfg.MODEL.compute_layout = compute_layout
-    cfg.freeze()
+def test_bert_small_cfg(compute_layout, ctx):
```

Comment:
Is it possible to use https://github.com/apache/incubator-mxnet/blob/8245fe2be7abd5c776b6ec6ecd2253232dd1efea/python/mxnet/test_utils.py#L64-L66 ?

Reply:
Yes. We can use this like `mx.context._current.set(mx.Context('gpu',0))` without `with` statement.

Reply:
This is an internal API and may change in the future. It may be better to use the `with` statement inside the `conftest.py`